### PR TITLE
Fix handling of non-ASCII characters in pl-rich-text-editor

### PR DESCRIPTION
**Pull request by jonatanschroeder**
Non-ASCII Unicode characters (accents, non-English alphabets, emojis, etc.) are properly encoded when submitted via `pl-rich-text-editor`, but the resulting text is not decoded properly when the result is shown again after submission. This should fix the issue (https://developer.mozilla.org/en-US/docs/Glossary/Base64#the_unicode_problem).

### CODE REVIEW

File: elements/pl-rich-text-editor/pl-rich-text-editor.js [hunk 1]
```diff
@@ -27,6 +27,9 @@ window.PLRTE = function(uuid, options) {
     quill.setContents(quill.clipboard.convert(contents));
     
     quill.on('text-change', function(_delta, _oldDelta, _source) {
-        inputElement.val(btoa(unescape(encodeURIComponent(quill.root.innerHTML))));
+        inputElement.val(btoa(he.encode(quill.root.innerHTML, {
+            allowUnsafeSymbols: true, // HTML tags should be kept
```
Comment (mwest1066) [inline elements/pl-rich-text-editor/pl-rich-text-editor.js:31]: This sounds bad, but I assume it's actually ok?

Comment (jonatanschroeder) [reply elements/pl-rich-text-editor/pl-rich-text-editor.js:31]: This means characters like `<`, `>`, `"`, etc. are not converted, which is what I actually want in this case.

Comment (jonatanschroeder) [reply elements/pl-rich-text-editor/pl-rich-text-editor.js:31]: https://github.com/mathiasbynens/he#allowunsafesymbols

> The default value for the allowUnsafeSymbols option is false. This means that characters that are unsafe for use in HTML content (&, <, >, ", ', and `) will be encoded. When set to true, only non-ASCII characters will be encoded.

Comment (mwest1066) [reply elements/pl-rich-text-editor/pl-rich-text-editor.js:31]: Ok, cool, thanks for explanation!

**Review by mwest1066 — APPROVED** [review]

**Pull request merged**